easybuild/easyconfigs/p/polymake/polymake-4.0r1-foss-2019b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'polymake'
+version = '4.0r1'
+
+homepage = 'https://polymake.org'
+description = """polymake is open source software for research in polyhedral geometry.
+It deals with polytopes, polyhedra and fans as well as simplicial complexes, matroids,
+graphs, tropical hypersurfaces, and other objects."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://polymake.org/lib/exe/fetch.php/download/']
+sources = [SOURCE_TAR_BZ2]
+checksums = ['06654c5b213e74d7ff521a4f52e446f46a54e52e7da795396b79dd8beead3000']
+
+dependencies = [
+    ('Ninja', '1.9.0'),
+    ('libxml2', '2.9.9'),
+    ('libxslt', '1.1.34'),
+    ('libreadline', '8.0'),
+    ('Perl', '5.30.0'),
+    ('SVG', '2.84', '-Perl-5.30.0'),
+    ('GMP', '6.1.2'),
+    ('MPFR', '4.0.2'),
+    ('Boost', '1.71.0'),
+    ('Java', '1.8', '', True),
+    ]
+
+# configopts = "--enable-libnuma=$EBROOTNUMACTL"
```

Comment:
@terjekv This commented out line can be removed?

Also, please add a custom `sanity_check_paths`. Here's a suggestion:

```python
sanity_check_paths = {
    'files': ['bin/polymake', 'bin/polymake-config', 'lib/libpolymake.%s' % SHLIB_EXT],
    'dirs': ['include/polymake', 'lib/polymake', 'share/polymake'],
}
```
```

Reply:
Sorted in https://github.com/easybuilders/easybuild-easyconfigs/pull/9904/commits/bcea366517b89bdf5c7518754ba87b5851ec7371.